### DISCUSSION TITLE
BT-1375-Procedure validation

### DISCUSSION THREAD

**Discussion by elerik** [OP-TED/eForms-SDK#545] [Q&A] [answered]
This is probably just about explanation (because I dont know whats are regulation requirements so i guess answer is non technical) 

I have this fields set 
![image](https://github.com/OP-TED/eForms-SDK/assets/11737559/a6fd8690-f438-417f-b13f-c029657bb62b)

But CVS return this error 
![image](https://github.com/OP-TED/eForms-SDK/assets/11737559/fb5f8409-b140-4f82-8f04-99e53f11b457)

As you can see i have only 1 LOT and only 1 GLO 

Ok. Lets fix it :) I append new field and select same value 
![image](https://github.com/OP-TED/eForms-SDK/assets/11737559/3ee0fe81-40dd-45cf-821e-e9549efbd28c)

And then problem solved. CVS is fine with it and and return no errors and is happy. 

Now question is. Is there something wrong with validation and it should be >= 1 and not just > 1 ? Because if i have only 1 GLO/Lot and select only 1 possibilities Validation should be ok with it. Or there is problem that I dont understand law and this is unrealistic test because in reality people will have more Lots/Glos ? :)

**Comment by YvesJo** [comment, marked as answer]
Hi,
To be able to define a Group of Lots, there is a need to already have 2 lots defined. 
Then the Group of lots itself should be composed of at least 2 lots.
Not all rules have currently been implemented to prevent situations like the one you proposed (add twice the same lot).
